state/state.go

```diff
@@ -926,6 +942,17 @@ func (s *State) DebugTransaction(ctx context.Context, transactionHash common.Has
 	}
 	endTime := time.Now()
 
+	// //save process batch response file
```

Comment:
This can be deleted

Reply:
Would you mind if I keep it commented in the test? This helped a lot extracting data to provide to the executor and protocol team in order to understand what was not working.